client/reader/following-stream/index.jsx

```diff
@@ -405,7 +405,7 @@ module.exports = React.createClass( {
 			hasNoPosts = store.isLastPage() && ( ( ! this.state.posts ) || this.state.posts.length === 0 ),
 			body;
 
-		if ( hasNoPosts ) {
+		if ( hasNoPosts || store.hasRecentError() ) {
```

Comment:
I was concerned about the same thing. Just made a update to that function. I wasn't sure why it was checking errors from the past minute and not just the last error.